stress/client/main.go

```diff
@@ -335,6 +337,6 @@ func main() {
 		close(stop)
```

Comment:
Optional for this PR but would be nice to fix up soon if we are reviving this:

More natural and simpler would be:

```go
ctx := context.Background
if *testDurationSecs > 0 {
	ctx, cancel = context.WithTimeout(ctx, *testDurationSecs * time.Second)
	defer cancel()
}
........
		performRPCs(ctx, g, conn, testSelector)
......

func performRPCs() {
....
	for ctx.Err() == nil {
		....
	}
}
```

Even better would be to _then_ pass `ctx` to `interop.DoBlah` but that probably affects other things too.  But it's following a bad practice and should be cleaned up one day.  I'll file an issue for that.

Reply:
(#6763 FYI)

Reply:
I need to chat with you a bit more about this, not changing in this PR.